ayushma/models/testsuite.py

```diff
@@ -30,9 +29,15 @@ class TestQuestion(BaseModel):
 
 
 class TestRun(BaseModel):
+    STATUS_CHOICES = [
+        ("running", "running"),
+        ("completed", "completed"),
+        ("canceled", "canceled"),
+        ("failed", "failed"),
+    ]
```

Comment:
Can we shift these to enums.py? Use Integer enums

Reply:
Yep, done.